Gemfile

```diff
@@ -0,0 +1,6 @@
+# By Jakub Kaspar 26/02/2020
```

Comment:
Can you remove these attributions from here and from the Dangerfile? There are no inline attributions elsewhere in the project, and the other config / automation / CI -type files do not have the MIT license attribution at the top.